tests/conftest.py

```diff
@@ -35,20 +38,36 @@ def setup_rand() -> Random:
     return Random(231)
 
 
+@fixture
+def setup_landscape():
+    return LandscapeConfiguration(holders=[ResourceHolder(str(uuid4()), 'holder1', IntervalGaussian(25, 0),
+                                                          materials=[Material('111', 'mat1', 100000)])])
+
+
+@fixture
+def setup_landscape_with_many_holders():
+    return LandscapeConfiguration(holders=[ResourceHolder(str(uuid4()), 'holder1', IntervalGaussian(100, 0),
+                                                          materials=[Material('111', 'mat1', 100000)]),
+                                           ResourceHolder(str(uuid4()), 'holder2', IntervalGaussian(100, 0),
+                                                          materials=[Material('222', 'mat2', 100000)])
+                                           ])
+
+
 @fixture(scope='session')
 def setup_simple_synthetic(setup_rand) -> SimpleSynthetic:
     return SimpleSynthetic(setup_rand)
 
 
 @fixture(params=[(graph_type, lag) for lag in [True, False]
                  for graph_type in ['manual',
-                                    'small plain synthetic', 'big plain synthetic',]],
-                                    # 'small advanced synthetic', 'big advanced synthetic']],
+                                    ]],
+         # 'small plain synthetic', 'big plain synthetic']],
+         # 'small advanced synthetic', 'big advanced synthetic']],
```

Comment:
REVERT. 
1) You should't disable synthetic graphs in tests
2) the code style in this place is correct